### PR TITLE
Fix competition form not rendering if the id hasn't been set yet

### PR DESCRIPTION
**Pull request by FinnIckler**

### CODE REVIEW

File: lib/microservices/registrations.rb [hunk 1]
```diff
@@ -85,6 +85,7 @@ def self.update_registration_payment(attendee_id, payment_id, iso_amount, curren
     # rubocop:enable Metrics/ParameterLists
 
     def self.competitor_count_by_competition(competition_id)
+      return 0 if competition_id.nil? || competition_id.empty?
```
Comment (gregorbg) [inline lib/microservices/registrations.rb:88]: alternative: `return 0 unless competition_id.present?`

Comment (FinnIckler) [reply lib/microservices/registrations.rb]: true, I could have also used `blank?`

**Comment by FinnIckler** [comment]
tests seems to fail because of some mailing list hotfix?

**Comment by FinnIckler** [comment]
merging as a hotfix as the failed tests are also on main and we want to allow people to create competitions again

**Pull request merged**